### PR TITLE
Matchfunding iteration

### PR DESCRIPTION
**Pull request by sauloperez**
## What?

This is the iteration on the landing page setting up for the upcoming matchfunding campaing. This includes:

- [ ] Better illustration. Now we have http://katuma.org/img/Katuma_esquema_platform_coop.jpg1
- [x] Update video
- [x] Make the video stand out
- [x] Give priority to the matchfunding (widget)
- [x] Change the call to action from enroll to donate and share
- [x] Section for each key concept: food sovereignty,  platform coop, free software
- [x] Discourse (journey) by actor: group, producer and consumer
- [x] Better top background image
- [x] Add call to action "Entra en Katuma" or something linking to the app
- [x] Track buttons in analytics
- [ ] Translate copies to all three languages

### CODE REVIEW

**Comment by sauloperez** [comment]
So far this looks like:

screencapture-file-home-pau-dev-katuma-landing-page-index-html-1519755262975

Obviously, the navbar needs to blend onto the background image.

**Comment by enricostano** [comment]
Me gusta, sólo podríamos buscar algún elemento que le quite look enterprise. Otra fuente?

Seguramente con las 3 fotos ganará color y vida. Cómo first iteration 😍👍

**Comment by sauloperez** [comment]
Ahora:

screencapture-file-home-pau-dev-katuma-landing-page-index-html-1519803753626

**Comment by enricostano** [comment]
I added `- [ ] Add call to action "Try the alpha" or something linking to the app` to the checklist, makes sense?

**Comment by sseerrggii** [comment]
@enricostano yo pondria un botón de "Entra en Katuma" a evitar ya el concepto alpha que lo queremos sacar pronto :)

**Comment by enricostano** [comment]
'Entra en Katuma' a lo mejor un bloque al fondo antes del footer

**Comment by sauloperez** [comment]
We keep moving! I'm going to finish up the content now.

screencapture-file-home-pau-dev-katuma-landing-page-index-html-1519829350873

**Comment by sauloperez** [comment]
We're done here! I'm going to translate the copies to all three languages.

screencapture-file-home-pau-dev-katuma-landing-page-index-html-1519838776194

**Comment by sauloperez** [comment]
I updated the last screenshot. That's how it looks like. I didn't manage to use i18next successfully 🙃 so I hardcoded the copies to Catalan, which the main language in our audience. We can work on it later today.

File: index.html [hunk 1]
```diff
@@ -3,155 +3,201 @@
...
       <div class="container">
         <div class="row header">
-          <h1 data-i18n="header.title">KATUMA</h1>
-          <h2 class="slogan" data-i18n="[html]header.slogan">Compra directamente a <span>Productoras agroecológicas de proximidad</span></h2>
+          <div class="col-lg-6">
+            <h1 class="slogan" data-i18n="[html]header.slogan">Compra directament a productes de proximitat</h1>
+            <p class="subtitle">
+            Consumeix productes de proximitat i participa d'un model sostenible amb beneficis socioeconòmics per a la comunitat.
+            </p>
+            <a onClick="ga('boton', 'click', 'Mathfunding2018-hero');" class="button negative-button" href="https://www.goteo.org/project/katuma">Fes-te mecenes</a>
```
Comment (enricostano) [inline index.html]: Typo o ke?

**Comment by sauloperez** [comment]
Live in katuma.org 🎉

**Comment by enricostano** [comment]
Le damos al merge de este y abrimos tareas pendientes a parte?

**Comment by sauloperez** [comment]
per mi ok

**Comment by enricostano** [comment]
Pues si puedes crear cards a Trello y merge.

**Pull request merged**